bin/rhsm-service

```diff
@@ -33,11 +33,12 @@ if __name__ == "__main__":
             objects.ConfigDBusObject,
             objects.RegisterDBusObject,
             objects.AttachDBusObject,
+            objects.ProductsDBusObject,
             objects.Main
         ]
         sys.exit(service_wrapper.main(sys.argv, object_classes=object_classes))
-    except Exception:
-        log.exception("DBus service startup failed")
+    except Exception as err:
+        log.exception("DBus service startup failed: %s" % err)
```

Comment:
`log.exception` will automatically include the stack information and exception message, so no need to add it.